js/models/BaseModel.js

```diff
@@ -116,7 +116,7 @@ export default class extends Model {
     // todo: will it break things if we unset a nested attribute?
     if (!opts.unset) {
       // let's work off of a clone since we modify attrs
-      attrs = JSON.parse(JSON.stringify(attrs));
+      // attrs = JSON.parse(JSON.stringify(attrs));
```

Comment:
It looks like taking this out is what breaks the moderatorInfo save in the settings.